supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystemValidatedRuntimeAttributesBuilder.scala

```diff
@@ -0,0 +1,74 @@
+package cromwell.backend.sfs
+
+import cromwell.backend.validation._
+
+/**
+  * Validates a collection of runtime attributes for a Shared File System (SFS) Backend.
+  *
+  * `default` returns the default set of attributes required to run an SFS:
+  * - `ContinueOnReturnCodeValidation.default`
+  * - `FailOnStderrValidation.default`
+  *
+  * Additional runtime attribute validations may be added by calling `withValidation` on the default.
+  *
+  * The SFS will also _always_ validate using the `DockerValidation`, but may end up warning the user that the attribute
+  * is unsupported by the backend implementation.
+  */
+object SharedFileSystemValidatedRuntimeAttributesBuilder {
+
+  lazy val default: SharedFileSystemValidatedRuntimeAttributesBuilder = {
+    val custom = Seq.empty
+    val required = Seq(ContinueOnReturnCodeValidation.default, FailOnStderrValidation.default)
+    val extra = Seq(DockerValidation.optional)
```

Comment:
In performing the hackery over on my branch I realized I don't understand the differences among these three groupings of validations.